### PR TITLE
add onelogin saml guide

### PR DESCRIPTION
**Pull request by MitchellGerdisch**

### CODE REVIEW

File: themes/default/content/docs/guides/saml/_index.md [hunk 1]
```diff
@@ -27,3 +28,4 @@ If you're looking to integrate Pulumi with your SAML 2.0 identity provider, refe
 - [Azure Active Directory]({{< relref "aad" >}})
 - [G Suite (Google)]({{< relref "gsuite" >}})
 - [Okta]({{< relref "okta" >}})
+- [OneLogin]({{< relref "onelogin" >}}
```
Comment (clstokes) [inline themes/default/content/docs/guides/saml/_index.md:31]:
```suggestion
- [OneLogin]({{< relref "onelogin" >}})
```

The link isn't rendering correctly. 
image

**Review by clstokes — CHANGES_REQUESTED** [review]
Just need to fix the link and then looks good to me!

**Comment by sean1588** [comment]
Hey Mitch, thanks for writing this up, this looks awesome! We were also able to verify these docs with one of our customers last week as well so seems to work well!

**Review by sean1588 — APPROVED** [review]

File: themes/default/content/docs/guides/saml/onelogin.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+## Prerequisites
+
+- [Single Sign-On]({{< relref "sso" >}})
+
+## Creating the OneLogin Application
+
+The first step is to create a new OneLogin Application for Pulumi SSO:
+
+- From the OneLogin Administration portal, go to the **Applications** page and click the **Add App** button.
```
Comment (davidwrede) [inline themes/default/content/docs/guides/saml/onelogin.md:25]: Please make these numbered steps.

File: themes/default/content/docs/guides/saml/onelogin.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+| Recipient                  | `https://api.pulumi.com/login/acmecorp/sso/saml/acs`      |
+| ACS Consumer URL Validator | `.*`                                                      |
+| ACS Consumer URL           | `https://api.pulumi.com/login/acmecorp/sso/saml/acs`      |
+| SAML initiator             | `OneLogin`                                                |
+| SAML nameID format         | `Email`                                                   |
+| SAML issuer type           | `Specific`                                                |
+| SAML signature element     | `Response`                                                |
+| SAML encryption method     | `TRIPLEDES-CBC`                                           |
+
+> **Important:** Do not change the value of the Name ID Format once your users have started using Pulumi---not even switching its value between `EmailAddress` or `Persistent`.
```
Comment (davidwrede) [inline themes/default/content/docs/guides/saml/onelogin.md:58]:
```suggestion
{{% notes type="warning" %}}
Do not change the value of the Name ID Format once your users have started using Pulumi---not even switching its value between `EmailAddress` or `Persistent`.
{{% /notes %}}
```

Can you add a bit more "why" to the end of this warning (e.g. what will happen if they change the value?)

File: themes/default/content/docs/guides/saml/onelogin.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+SAML application. To do this, you need to obtain the IDP metadata document from OneLogin and then provide
+it to Pulumi.
+
+First, navigate to the OneLogin Application you created above and click the **More Actions** drop down menu button and select _SAML Metadata_ to download the metadata XML file.
+
+![Get Metadata](/images/docs/reference/service/saml-onelogin/onelogin-get-metadata.png)
+
+- Open the file and copy the entire block of XML text in your clipboard
+- Open the Pulumi Console and navigate to your SAML organization.
+- Click the **Settings** tab, and then select **SAML SSO**.
```
Comment (davidwrede) [inline themes/default/content/docs/guides/saml/onelogin.md:94]: Please make these numbered steps.

**Review by davidwrede — CHANGES_REQUESTED** [review]
Just a few things to clean up and then this should be gtg.

File: themes/default/content/docs/guides/saml/onelogin.md (hunk 2 above)
Comment (MitchellGerdisch) [reply themes/default/content/docs/guides/saml/onelogin.md]: Done

File: themes/default/content/docs/guides/saml/onelogin.md (hunk 4 above)
Comment (MitchellGerdisch) [reply themes/default/content/docs/guides/saml/onelogin.md]: Done

File: themes/default/content/docs/guides/saml/onelogin.md (hunk 3 above)
Comment (MitchellGerdisch) [reply themes/default/content/docs/guides/saml/onelogin.md:58]: TBH, I don't know - this is text I copied from other SSO SAML docs pages (e.g. Okta).

Comment (davidwrede) [reply themes/default/content/docs/guides/saml/onelogin.md:58]: Gotcha. We can leave it as is for now.

**Review by davidwrede — APPROVED** [review]

**Review by clstokes — APPROVED** [review]

**Pull request merged**